color_in_sequence.py

```diff
@@ -0,0 +1,130 @@
+
+# library for waiting for network operations to finish
+import asyncio 
+
+# Just for knowing what types you can enter in your IDE
+from typing import List, Tuple 
+
+# Custom util I made for saving bulbs.
+# This function gets you the lights directly via a name
+from utils.bulb_groups import get_wiz_light_from_group
+
+# PilotBuilder makes colors and settings
+# Wizlight is how you make calls to a light
+from pywizlight import PilotBuilder, wizlight
+
+# You know this one
+from time import sleep
+
+# continually iterates through an array
+from itertools import cycle, islice
+
+
+assortment_of_color_rgbs = [
```

Comment:
```suggestion
ASSORTMENT_OF_COLOR_RGBS = [
```

This one is my bad, but common python syntax is to make constants upper case. Just like C.